staging/src/k8s.io/apiserver/pkg/endpoints/handlers/apply/typeconverter.go

```diff
@@ -62,6 +64,16 @@ func (c *typeConverter) ObjectToTyped(obj runtime.Object) (typed.TypedValue, err
 	return t.FromUnstructured(u)
 }
 
+func (c *typeConverter) YAMLToTyped(from []byte) (typed.TypedValue, error) {
```

Comment:
That's probably testable

Reply:
Can you check my last commit and see if that's enough?
Was the easiest way i saw to quickly test if it works. It at least tests against the other functions.